app/public/js/profile_save_prompt.js

```diff
@@ -1,12 +1,9 @@
 $(window).on('beforeunload', function(e) {
 	if ( submitted ) {
 		return;
-	}
+	}	
```

Comment:
remove this unneeded space or tab after the }.  The line doesn't make a useful change for other developers yet it will interfere with "git blame" on this file showing useful information about when and why the line was last changed.